language-design.md

```diff
@@ -49,6 +163,22 @@ In what way is an API a language?
 
 **Response**
 
+An API is similar to a language stripped of its fluency. It consists of 
```

Comment:
I think "a language stripped of its fluency" is a great explanation of an API, especially with the rest of your response